OpenDreamShared/Compiler/DM/DMParser.cs

```diff
@@ -921,6 +921,14 @@ public class DMParser : Parser<Token> {
 
                 if (Check(TokenType.DM_Comma)) {
                     Whitespace();
+
+                    if (Check(TokenType.DM_IndeterminateArgs) && !Check(TokenType.DM_Comma))
```

Comment:
This breaks if there is whitespace after the IndeterminateArgs token (`proc/example(a, ... , b)`). Rewriting the loop like this would probably be best:
```cs
public DMASTDefinitionParameter[] DefinitionParameters() {
    List<DMASTDefinitionParameter> parameters = new();
    DMASTDefinitionParameter parameter = DefinitionParameter();

    if (parameter != null) parameters.Add(parameter);
    while (Check(TokenType.DM_Comma)) {
        Whitespace();

        parameter = DefinitionParameter();
        if (parameter != null) {
            parameters.Add(parameter);
        } else if (!Check(TokenType.DM_IndeterminateArgs)) {
            Error("Expected parameter definition");
        }
    }

    return parameters.ToArray();
}
```